test/k8s/assertion_helpers.go

```diff
@@ -15,18 +15,6 @@ import (
 
 var longTimeout = 10 * time.Minute
 
-// ExpectKubeDNSReady is a wrapper around helpers/WaitKubeDNS. It asserts that
-// the error returned by that function is nil.
-func ExpectKubeDNSReady(vm *helpers.Kubectl) {
-	By("Waiting for kube-dns to be ready")
-	err := vm.WaitKubeDNS()
```

Comment:
Looks like this was the only call site of (*Kubectl).WaitKubeDNS, so it could be removed from `test/helpers/kubectl.go` as well.